routes/web.php

```diff
@@ -41,14 +41,20 @@
     //route for announcements
     Route::get('/dashboard/announcements', [HubController::class, 'getAnnouncements'])->name('announcements');
 
-    //route for my profile in personal hun
+    //route for my profile in personal hub
     Route::get('/dashboard/profile', [HubController::class, 'getProfileInfo'])->name('my-profile');
 
     //route for personal programme
-    Route::get('/dashboard/programme', [HubController::class, 'getProgramme'])->name('my-programme');
+    Route::get('/dashboard/programme', [HubController::class, 'personalProgramme'])->name('my-programme');
```

Comment:
When this gets merged: I have renamed the routes to 'myconference/...' in PR #149 . Keep that in mind.